docs/source/supergraphs.md

```diff
@@ -74,3 +74,16 @@ rover supergraph compose --config ./supergraph.yaml > prod-schema.graphql
 ```
 
 > For more on passing values via `stdout`, see [Using `stdout`](./conventions#using-stdout).
+
+#### Gateway compatibility
+
+The `rover supergraph compose` command produces a supergraph by utilizing the composition functions of the [`@apollo/federation`](https://www.apollographql.com/docs/federation/api/apollo-federation/) package.  As that library is still in pre-1.0 releases (as are Rover and Apollo Gateway), some updates to Rover may result in a supergraph with new functionality and therefore necessitate complimentary updates to Apollo Gateway.
+
+Apollo Gateway will fail to start-up if it's provided with a supergraph that it doesn't support, so it's recommended to test launching Apollo Gateway with the supergraph it will be ultimately be deployed with in a CI pipeline to ensure compatible versions.
+
+We aim to reduce the frequency at which these paired updates are necessary by making supergraph additions backwards-compatible.  We will note changes which require corresponding Apollo Gateway updates  clearly in the Rover _Release Notes_ as well as updating the compatibility table below.
```

Comment:
"backwards-compatible" -> "backward compatible" (there's no definite right format for this term, but this is the one we standardize on)

Reply:
"which require" -> "that require"

"as well as updating" -> ", and we'll also update the following compatibility table:"